<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>409</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string notr="true">&amp;Region</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="nbtBtn">
            <property name="text">
             <string>&amp;NBT File Reader</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mrBtn">
            <property name="text">
             <string>&amp;Manage Region</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="mrBtn_2">
            <property name="text">
             <string>M&amp;odify Region</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="brBtn">
            <property name="text">
             <string>&amp;Block reader</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Shortcuts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Utility</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="configBtn">
            <property name="text">
             <string>&amp;Configure</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scriptBtn">
            <property name="text">
             <string>&amp;Run Script</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="debugBtn">
            <property name="text">
             <string notr="true">Debug</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuIcons">
     <property name="title">
      <string>Icons</string>
     </property>
    </widget>
    <addaction name="menuIcons"/>
   </widget>
   <addaction name="menu_Tools"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actiontest_2">
   <property name="text">
    <string>test\</string>
   </property>
  </action>
  <action name="actiontetst">
   <property name="text">
    <string>tetst</string>
   </property>
  </action>
  <action name="actiontest_3">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
